src/Microsoft.DotNet.Cli.Utils/Env.cs

```diff
@@ -33,5 +33,29 @@ public static string GetCommandPathFromRootPath(string rootPath, string commandN
         {
             return _environment.GetCommandPathFromRootPath(rootPath, commandName, extensions);
         }
+
+        public static bool GetBool(string name, bool defaultValue = false)
```

Comment:
Please add this method to the EnvironmentProvider.cs and forward the call like other methods in this class. Allows for injection of Environment in unit testing.